packages/xo-server/src/xapi-object-to-xo.js

```diff
@@ -76,6 +76,7 @@ const TRANSFORMS = {
         cores: cpuInfo && +cpuInfo.cpu_count,
         sockets: cpuInfo && +cpuInfo.socket_count,
       },
+      zstd_supported: obj.restrictions.restrict_zstd_export === 'false',
```

Comment:
The `zstd` is supported when `restrict_zstd_export` is `false` ?

Reply:
Yes, because it not have a restriction on `zstd` and if it supported the property will be defined.